packages/docusaurus-plugin-content-docs/index.js

```diff
@@ -0,0 +1,207 @@
+/**
+ * Copyright (c) 2017-present, Facebook, Inc.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+const path = require('path');
+const globby = require('globby');
+const {getSubFolder, idx, normalizeUrl} = require('@docusaurus/utils');
+const createOrder = require('./src/order');
+const loadSidebars = require('./src/sidebars');
+const processMetadata = require('./src/metadata');
+
+const DEFAULT_OPTIONS = {
+  metadataKey: 'docsMetadata',
+  metadataFileName: 'docsMetadata.json',
+  path: 'docs', // Path to data on filesystem, relative to site dir.
```

Comment:
Will it be better to have it as '../docs' (following v1's behavior) ?

But I personally like that the docs folder is inside the siteDir. But this is worth to think about

Reply:
I thought about this too, but I think we should design for best practices that's forward looking. It's good enough that we provide a backward-compatible way.